test/integration/ads_integration_test.cc

```diff
@@ -2374,4 +2377,106 @@ TEST_P(XdsTpAdsIntegrationTest, EdsAlternatingLedsUsage) {
   makeSingleRequest();
 }
 
+// A separate test fixture for running xdstp tests on the UnifiedSotw protocol.
+// Most of the XdsTpAdsIntegrationTest tests use LEDS, which is not supported
+// for State-of-the-World yet.
+class UnifiedSotwXdsTpAdsIntegrationTest : public XdsTpAdsIntegrationTest {};
+
+INSTANTIATE_TEST_SUITE_P(
+    UnifiedSotw, UnifiedSotwXdsTpAdsIntegrationTest,
+    testing::Combine(testing::ValuesIn(TestEnvironment::getIpVersionsForTest()),
+                     // There should be no variation across clients.
+                     testing::Values(Grpc::ClientType::EnvoyGrpc),
+                     testing::Values(Grpc::SotwOrDelta::UnifiedSotw),
+                     testing::Values(OldDssOrNewDss::Old, OldDssOrNewDss::New)));
+
+TEST_P(UnifiedSotwXdsTpAdsIntegrationTest, Basic) {
```

Comment:
Is this a copy-paste of the Basic test?
If so, and assuming that more tests will be added for both of SotW and Delta, I suggest merging the two (test-parameter) and skipping the LEDS tests in SotW mode.